.github/workflows/integration_test.yaml

```diff
@@ -0,0 +1,28 @@
+name: Integration tests
+
+on:
+  pull_request:
+
+jobs:
+  integration-test-microk8s:
+    name: Integration tests (microk8s)
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup operator environment
+        uses: charmed-kubernetes/actions-operator@main
+        with:
+          juju-channel: 3.1/stable
+          provider: microk8s
+          microk8s-addons: "ingress storage dns rbac registry"
+          channel: 1.25-strict/stable
+      - name: Run integration tests
+        # set a predictable model name so it can be consumed by charm-logdump-action
+        run: tox -e integration -- --model testing
+      - name: Dump logs
+        uses: canonical/charm-logdump-action@main
```

Comment:
This action is now available on the [v2.6.0 release](https://github.com/canonical/charming-actions/releases/tag/2.6.0). 

As a side note for the future, I recommend setting up renovatebot to get your actions and deps pinned to exact commits or versions and have renovate update them automatically. 

This next bit is probably worth a CSShare, but the observability charms and identity charms have some nice CI setup to automatically merge renovatebot PRs that pass unit/integration tests.
I've adapted Hydra's [renovate config](https://github.com/canonical/hydra-operator/blob/main/renovate.json) for example in JIMM. Then you can setup [some](https://github.com/canonical/hydra-operator/blob/main/.github/workflows/auto-approver.yaml) cool [automation](https://docs.renovatebot.com/key-concepts/automerge/) to get those PRs merged without user intervention.

Reply:
To accommodate for the new integration test of running an Airbyte sync, I actually had to switch to the integration tests workflow offered by the IS Devops team to be able to use larger self-hosted Github runners.